packages/web-components/tests/e2e-storybook/cypress/integration/masthead/cloud/cloud-masthead.e2e.js

```diff
@@ -44,6 +44,28 @@ describe('dds-masthead | cloud platform (desktop)', () => {
       });
   });
 
+  it('should support custom url for IBM logo', () => {
+    cy.intercept(`https://1.www.s81c.com/common/carbon-for-ibm-dotcom/translations/cloud-masthead/*`, {
+      fixture: 'translation-custom-logo.json',
+    }).as('endpointInterceptor');
```

Comment:
Nit: I don't think we need to alias this interceptor as we never reference it again

Reply:
Fair point. I left it in there since it is helpful for identifying successful intercepts in the Cypress logs. My preference is to keep it as it's not hurting anything, but I'm happy to defer if you feel strongly otherwise.

Reply:
Nope, that's rationale enough for me!